action.yml

```diff
@@ -31,9 +31,24 @@ runs:
         version: ${{ inputs.ghc-version }}
         url: "https://github.com/freckle/weeder-action/releases/download/Binaries/{name}-{version}-{os}-{arch}.{ext}"
 
+    - id: weeder-version
+      name: Record weeder version
+      shell: bash
+      run: |
+        weeder --version | sed '/^weeder version \(.*\)$/!d; s//result=\1/' >>"$GITHUB_OUTPUT"
+
+    - id: weeder-check
+      name: Compare weeder version with v2.7
+      uses: pbrisbin/semver-compare-action@master
```

Comment:
Looks like this fork just removes warnings about an outdated version of Node?

Reply:
It does. The PR sits open with no response for weeks now though, so I might be just taking over for myself...